pkg/config/config.go

```diff
@@ -539,6 +543,7 @@ type Config struct {
 	Linters         Linters
 	Issues          Issues
 	Severity        Severity
+	Version         Version
```

Comment:
so this one will come in .golangci.yaml file as well ?

Reply:
yes, it does.
```shell
20-08-18 10:28 % ./golangci-lint version
golangci-lint has version (devel) built from (unknown, mod sum: "") on (unknown)


20-08-18 10:28 % git diff
diff --git a/.golangci.yml b/.golangci.yml
index 5e9e1e7..fa899f6 100644
--- a/.golangci.yml
+++ b/.golangci.yml
@@ -146,3 +146,6 @@ service:
   golangci-lint-version: 1.23.x # use the fixed version to not introduce new linters unexpectedly
   prepare:
     - echo "here I can run custom commands, but no preparation needed for this repo"
+
+version:
+  format: json
20-08-18 10:28 % ./golangci-lint version
{"version":"(devel)","commit":"(unknown, mod sum: \"\")","date":"(unknown)"}
```

Reply:
My original concern was that if this should be part of configuration file. Then, I think it's not much a big deal as long as we don't break declarative understanding of config file (e.g. it will be very weird to specify actual version in config file).

Reply:
oh no, we don't specify a version in the config file, just the output format